e2e/03-intraledger-flow.e2e.spec.ts

```diff
@@ -107,28 +107,28 @@ describe("Username Payment Flow", () => {
     await clickButton(LL.SendBitcoinConfirmationScreen.title())
   })
 
-  it("Clicks on not enjoying app", async () => {
-    await browser.pause(3000)
-    const contexts = await browser.getContexts()
-    const nativeContext = contexts.find((context) =>
-      context.toString().toLowerCase().includes("native"),
-    )
-    await browser.pause(3000)
-    if (nativeContext) {
-      await browser.switchContext(nativeContext.toString())
-    }
+  if (process.env.E2E_DEVICE === "android") {
```

Comment:
why is that needed?

Reply:
fyi, tests should typically not be skipped like that. it's not a good practice. at a minimum, we should document why we are skipping it if that's necessary

Reply:
also, why may it pass on other tests, like here? https://github.com/GaloyMoney/galoy-mobile/pull/2499 is it flacky or always not passing?

Reply:
because for ios we have autodismiss alert, so not required for ios
first I thought this was causing the issue, but it was the useEffect

Reply:
> because for ios we have autodismiss alert, so not required for ios

no anymore (afaik)

Reply:
Yes, I have added it in the feedback flow pr